build.sbt

```diff
@@ -33,11 +37,16 @@ lazy val cli = project
     assembly / mainClass       := Some("aqua.AquaCli"),
     assembly / assemblyJarName := "aqua-cli-" + version.value + ".jar",
     libraryDependencies ++= Seq(
-      "com.monovore"  %% "decline"        % declineV,
-      "com.monovore"  %% "decline-effect" % declineV,
-      "org.typelevel" %% "cats-effect"    % catsEffectV,
-      "co.fs2"        %% "fs2-core"       % fs2V,
-      "co.fs2"        %% "fs2-io"         % fs2V
+      "com.monovore"       %% "decline"            % declineV,
+      "com.monovore"       %% "decline-effect"     % declineV,
+      "org.typelevel"      %% "cats-effect"        % catsEffectV,
+      "co.fs2"             %% "fs2-core"           % fs2V,
+      "co.fs2"             %% "fs2-io"             % fs2V,
+      "org.typelevel"      %% "log4cats-slf4j"     % "2.1.1",
```

Comment:
Move versions to constants plz, especially where they correspond to each other (log4cats-core, log4cats-slf4j).